app/services/audited_operations_service.rb

```diff
@@ -77,10 +77,12 @@ def self.access_shared_birth_record(logged_identity:, share:)
     raise ArgumentError, 'share cannot be nil' if share.nil?
     raise ArgumentError, 'logged_identity cannot be nil' if logged_identity.nil?
     raise ShareRevokedError if share.revoked?
-    raise UnauthorisedAccessRequestError if logged_identity != share.recipient
+
+    # Authorization is happening with Pundit
+    # raise UnauthorisedAccessRequestError unless Pundit::SharePolicy.new(logged_identity, share).show?
```

Comment:
🙋‍♀️I'm not sure what to do here - authorization is happening with Pundit on the controller level but it can't hurt to have it here too - but I wouldn't want to reimplement the rule that Pundit is using. Should there be a `user.can_view_share?` method, or should I bring in the same Pundit method here, or can I just leave authorization out of this service?

Reply:
yeah, i think the auth should all be in pundit.